Packs/Okta/Integrations/Okta_v2/Okta_v2.py

```diff
@@ -496,6 +496,33 @@ def clear_user_sessions(self, user_id):
             resp_type='text'
         )
 
+    def get_zone(self, args, zoneID):
+        uri = f'zones/{zoneID}'
+        return self._http_request(
+            method='GET',
+            url_suffix=uri,
+            resp_type='text'
+        )
+
+    def list_zones(self, args):
+        uri = f'zones'
```

Comment:
```suggestion
        uri = 'zones'
```
no need for `f` if there is no formatting